app/routes/aldersovergang._index.tsx

```diff
@@ -54,29 +60,41 @@ export default function BatchOpprett_index() {
     <div>
       <h1>Start aldersovergang</h1>
       <Form action="bpen005" method="POST">
-        <div style={{ display: 'inline-block' }}>
-          <label>Behandlingsmåned</label>
-          <br />
-          <input
-            defaultValue={denneBehandlingsmaneden}
-            aria-label="Behandlingsmåned"
-            name="behandlingsmaned"
-            type="number"
-            placeholder="Behandlingsmåned"
-          />
-        </div>
         <br />
-        <div style={{ display: 'inline-block' }}>
-          <Select
-            label="Begrenset utplukk"
-            size={'small'}
-            name={'begrensetUtplukk'}
-            defaultValue={'false'}
-          >
-            <option value="true">Ja</option>
-            <option value="false">Nei</option>
-          </Select>
-        </div>
+        <label><b>Behandlingsmåned</b></label>
+        <br />
+        <input
+          name="behandlingsmaned"
+          defaultValue={denneBehandlingsmaneden}
+          type="number"
+          placeholder="Behandlingsmåned"
+        />
+
+        <br />
```

Comment:
Tror dere ønsker å bruke en HStack, VStack eller liknende fra Aksel i stedet for <br> 😊

Reply:
Tar endringene i styling i denne: https://github.com/navikt/pensjon-verdande/pull/147